# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-core/LICENSE.MIT
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/wgpu-core/LICENSE.MIT
--- a/third_party/rust/wgpu-core/LICENSE.MIT	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/wgpu-core/LICENSE.MIT	Tue Oct 07 17:00:40 2025 +0000
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 The gfx-rs developers
+Copyright (c) 2025 The gfx-rs developers
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal